NitroxServer/GameLogic/Bases/BaseManager.cs

```diff
@@ -98,19 +156,31 @@ public void BasePieceDeconstructionBegin(NitroxId id)
             lock (completedBasePieceHistory)
             {
                 basePiece = completedBasePieceHistory.Find(piece => piece.Id == id);
-
+                if (basePiece.ConstructionAmount < 1f && partiallyConstructedPiecesById.ContainsKey(basePiece.Id))
```

Comment:
This statement looks like it would break deconstruction.  It will only remove the base piece from completed list (and set the construction amount/flag) if the construction amount was less than the completed value.

Reply:
thankyou

Reply:
I meant to set that to false